config/tileset/aerial.json

```diff
@@ -436,14 +436,6 @@
       "category": "Rural Aerial Photos",
       "minZoom": 13
     },
-    {
-      "2193": "s3://linz-basemaps/2193/timaru_urban_2012-2013_0-075m_RGBA/01F6P1XAGNDZVM0DH9JCFSE7QB",
-      "3857": "s3://linz-basemaps/3857/timaru_urban_2012-2013_0-075m_RGBA/01ED83HWB82K047N1KYK9A5SKD",
-      "name": "timaru-urban-2012-2013-0.075m",
```

Comment:
There is no individual config created for `timaru-urban-2012-2013-0.075m`. Or we already have one.

Reply:
Yea we already have one.